categories/basis.tex

```diff
@@ -21,276 +21,709 @@
 %
 \titlecard{Basis}%
 %
-\whitecard{Basis}{Ackerbau.}%
 \whitecard{Basis}{Aktives Zuhören.}%
+% engl. "Active listening"
 \whitecard{Basis}{Alkoholismus.}%
-\whitecard{Basis}{Alten Leuten winken.}%
-\whitecard{Basis}{An Dingen lecken, um sie für sich zu beanspruchen.}%
-\whitecard{Basis}{Anfangen, zu singen und zu tanzen.}%
+% engl. "Alcoholism"
+\whitecard{Basis}{Alten Leuten zuzwinkern.}%
+% before: zuwinken
```

Comment:
Remove the lines with `% before`. The new translations are much better and the git history should be enough if anybody cares about the old ones.